lib/irc-handler.js

```diff
@@ -174,33 +174,48 @@ var IRC_EVENTS = {
   },
   join: function join(context, channel, nick, message) {
     context.logger.verbose('IRC: %s joined %s', nick, channel);
-    HELPERS.sendChannelStatus(context, channel, nick + ' joined');
+    // Only send the JOIN to slack if it isn't ignored.
+    if (context.irc.options.hasOwnProperty('mute') && !context.irc.options.mute.includes('join')) {
```

Comment:
I don't think `Array.includes` is supported by older versions of node. Might want to use something else here.